tools/scripts/platform/xilinx/util.tcl

```diff
@@ -242,7 +242,11 @@ proc upload {} {
 	set cpu [_get_processor]
 
 	# Connect to the fpga
-	connect
+  if { [info exists ::env(XSCT_REMOTE_HOST)] || [info exists ::env(XSCT_REMOTE_PORT)] } {
+    connect -host "$::env(XSCT_REMOTE_HOST)" -port "$::env(XSCT_REMOTE_PORT)"
```

Comment:
this seems like a cool feature but we will only be known to you after merging, unless documented somewhere, like the no-OS build guide in the Xilinx sections

Reply:
what would the flow be like? something like this?
```
export XSCT_REMOTE_HOST=11.22.33.44
export XSCT_REMOTE_PORT=5566
make run
```

Reply:
> what would the flow be like? something like this?
> 
> ```
> export XSCT_REMOTE_HOST=11.22.33.44
> export XSCT_REMOTE_PORT=5566
> make run
> ```

Yes exactly